src/components/RoundButton/RoundButton.jsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from "prop-types";
+import "./RoundButton.css";
+
+const RoundButton = ({ onClick, children, variant, isActive }) => {
```

Comment:
Methods at the end and values first:

```suggestion
const RoundButton = ({ children, variant, isActive, onClick }) => {
```